x/authenticator/integration_test.go

```diff
@@ -740,163 +642,3 @@ func (s *AuthenticatorSuite) TestAuthenticatorAddRemove() {
 	err = s.app.AuthenticatorKeeper.RemoveAuthenticator(s.chainA.GetContext(), accountAddr, 2)
 	s.Require().NoError(err, "Failed to remove authenticator")
 }
-
-func (s *AuthenticatorSuite) TestSpendWithinLimitWithAuthzTableTest() {
```

Comment:
oh man! this table test! it was so beautiful!

Reply:
:smiling_face_with_tear: